src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -111,8 +114,13 @@ const TabbedLayout: FunctionComponent<TabbedLayoutProps> = ({
                                             { subtitle }
                                         </Typography>
                                     </Box>
-                                    <Box>
-                                        { actionButtons }
+                                    <Box display="flex" flexDirection="row">
+                                        <Box>
+                                            { actionButtons }
+                                        </Box>
+                                        { !!ellipsisMenu?.length && (
+                                            <EllipsisMenu items={ ellipsisMenu }/>
+                                        ) }
```

Comment:
I don't love that we now have two implementations of the same thing here, since the action buttons hold the existing menues. I don't know if we should solve it in this PR... I guess the solution isn't to remove action buttons, because they could still be used for the high impact buttons in the title area. But that we should move all existing menues to this method.

Reply:
I agree, but I figured it was outside the scope of this PR. I was thinking we could create separate issues, or even defer such changes until the next time we actually work on those menus. I should have communicated that somehow.

Maybe we should add a section to the PR template called "Deferred tasks" or "Out of scope" or something?

Reply:
Yea maybe we should...

Reply:
I have created separate issue #522 for this